### DISCUSSION TITLE
admin password setting is missing

### DISCUSSION THREAD

**Discussion by swubian** [hiddify/Hiddify-Manager#855] [Q&A سوالات رایج]
Hi.
how can I change admin password? it is not located in admin section of web panel setting as you can see in attached image. also I can't find netdata under general setting. is there any way for me to change admin password?
![2023-04-29_01-55-17](https://user-images.githubusercontent.com/128418843/235264024-2f1bebf3-764c-41a0-a1f2-b4444d6e79d7.jpg)

**Comment by josephjul40** [comment]
even me  i ask how to change admin link or password
